### ISSUE TITLE
Node.js  Build WorkGroup Meeting 2020-08-18

### ISSUE THREAD

**Issue by mhdawson** [nodejs/build#2409]
## Time

**UTC Tue 18-Aug-2020 20:00 (08:00 PM)**:

| Timezone      | Date/Time             |
|---------------|-----------------------|
| US / Pacific  | Tue 18-Aug-2020 13:00 (01:00 PM) |
| US / Mountain | Tue 18-Aug-2020 14:00 (02:00 PM) |
| US / Central  | Tue 18-Aug-2020 15:00 (03:00 PM) |
| US / Eastern  | Tue 18-Aug-2020 16:00 (04:00 PM) |
| London        | Tue 18-Aug-2020 21:00 (09:00 PM) |
| Amsterdam     | Tue 18-Aug-2020 22:00 (10:00 PM) |
| Moscow        | Tue 18-Aug-2020 23:00 (11:00 PM) |
| Chennai       | Wed 19-Aug-2020 01:30 (01:30 AM) |
| Hangzhou      | Wed 19-Aug-2020 04:00 (04:00 AM) |
| Tokyo         | Wed 19-Aug-2020 05:00 (05:00 AM) |
| Sydney        | Wed 19-Aug-2020 06:00 (06:00 AM) |

Or in your local time:
* http://www.timeanddate.com/worldclock/fixedtime.html?msg=Node.js+Foundation+Build%20WorkGroup+Meeting+2020-08-18&iso=20200818T20
* or http://www.wolframalpha.com/input/?i=08PM+UTC%2C+Aug+18%2C+2020+in+local+time

## Links

* Minutes Google Doc: <https://docs.google.com/document/d/1DJGXwQ_BCXqU_b8WVZf2jwDbTTwFitSh3UYOeu1eY1k/edit>

## Agenda

Extracted from **build-agenda** labelled issues and pull requests from the **nodejs org** prior to the meeting.

## Invited

* Build team: @nodejs/build

## Notes

The agenda comes from issues labelled with `build-agenda` across **all of the repositories in the nodejs org**. Please label any additional issues that should be on the agenda before the meeting starts.

## Joining the meeting

* link for participants: https://zoom.us/j/715960833
* For those who just want to watch: We stream our conference call straight to YouTube so anyone can listen to it live, it should start playing at https://www.youtube.com/c/nodejs+foundation/live when we turn it on. There's usually a short cat-herding time at the start of the meeting and then occasionally we have some quick private business to attend to before we can start recording & streaming. So be patient and it should show up.
* youtube admin page: https://www.youtube.com/my_live_events?filter=scheduled

**Comment by jbergstroem** [comment]
So, attendance looks to be two. Does this mean we postpone? Also, there was a doodle about changing the time; did that come into place?

**Comment by richardlau** [comment]
If the meeting goes ahead I can join. 

@AshCripps was organizing the time slot change. He is on vacation this week so the time change hasn't come into effect.

**Comment by jbergstroem** [comment]
I just got called into something I cannot move; I will unfortunately have to pass.

**Comment by richardlau** [comment]
@mhdawson if it's just us two, with nothing on the agenda, I'd suggest cancelling.

**Comment by mhdawson** [comment]
@richardlau agreed, will close and remove from the calendar
